great_expectations/expectations/core/expect_column_proportion_of_unique_values_to_be_between.py

```diff
@@ -36,9 +37,29 @@
     )
     from great_expectations.render.renderer_configuration import AddParamArgs
 
+EXPECTATION_SHORT_DESCRIPTION = (
+    "Expect the proportion of unique values to be between a minimum value and a maximum value."
+)
+MIN_VALUE_DESCRIPTION = (
+    "The minimum proportion of unique values.(Proportions are on the range 0 to 1)"
+)
+MAX_VALUE_DESCRIPTION = (
+    "The maximum proportion of unique values.(Proportions are on the range 0 to 1)"
```

Comment:
"The maximum proportion of unique values (proportions are on the range 0 to 1)."